### PR TITLE
Fix missing context menu on right click in the community column + tests

### LINKED ISSUE #6609: Context menu missing + regression test
# Bug Report

## Steps to reproduce

1. go to a community you have admin rights
2. right click on the community sidebar

expected: context menu to create a channel, category
actual: nothing happens

This bugfix should include a squish test.

### PR DESCRIPTION
**Pull request by jrainville**
Fixes #6609

The menu was lost because it used to be set as part of the ScrollView background, but it got changed to  a Flickable as part of the fix to the scrolls being not smooth.
Anyway, background doesn't exist on Flickables, so I removed it and moved the MouseArea to the CommunityColumnView instead.

Second commit adds integration tests

StatusQ PR: https://github.com/status-im/StatusQ/pull/802

### CODE REVIEW

**Comment by jrainville** [comment]
@noelia-santos you are the tester for this one

File: ui/app/AppLayouts/Chat/views/CommunityColumnView.qml [hunk 1]
```diff
@@ -496,30 +509,11 @@ Item {
...
-                    }
-
-                    StatusMenuItem {
-                        objectName: "createCommunityCategoryBtn"
-                        text: qsTr("Create category")
-                        icon.name: "channel-category"
-                        onTriggered: Global.openPopup(createCategoryPopup)
+                        adminPopupMenu.showInviteButton = false
+                        adminPopupMenu.popup()
+                        adminPopupMenu.y = Qt.binding(function () {
```
Comment (micieslak) [inline ui/app/AppLayouts/Chat/views/CommunityColumnView.qml:514]: could be slightly more concise with the new notation:
```
                        adminPopupMenu.y = Qt.binding(() =>
                            root.height - adminPopupMenu.height - createChannelOrCategoryBtn.height - 20
                        )
```

File: ui/app/AppLayouts/Chat/views/CommunityColumnView.qml [hunk 2]
```diff
@@ -100,6 +114,45 @@ Item {
         }
     }
 
+    StatusPopupMenu {
+        property bool showInviteButton: false
+
+        id: adminPopupMenu
```
Comment (micieslak) [inline ui/app/AppLayouts/Chat/views/CommunityColumnView.qml:120]: please keep id first

**Review by micieslak — APPROVED** [review]
LGTM, but 2 small suggestions added

File: test/ui-test/testSuites/suite_status/tst_communityFlows/test.feature [hunk 3]
```diff
@@ -34,21 +34,23 @@ Feature: Status Desktop community
     Scenario Outline: Admin creates a community channel
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/tst_communityFlows/test.feature:34]: Some identation issues in the previous examples (User creates a community) scenario

File: test/ui-test/testSuites/suite_status/tst_communityFlows/test.feature [hunk 4]
```diff
@@ -34,21 +34,23 @@ Feature: Status Desktop community
...
+            | test-channel2   | Community channel description tested 2      | right_click_menu |
 
     Scenario Outline: Admin edits a community channel
         When the user creates a community named myCommunity, with description My community description, intro Community Intro and outro Community Outro
         Then the user lands on the community named myCommunity
-        When the admin creates a community channel named test-channel, with description My description
+        When the admin creates a community channel named test-channel, with description My description with the method bottom_menu
         Then the user lands on the community channel named test-channel
         When the admin edits a community channel named <community_channel_name> to the name <new_community_channel_name>
         Then the user lands on the community channel named <new_community_channel_name>
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/tst_communityFlows/test.feature:51]: I'm thinking here we can add more validations, maybe in a separated task. I did some of them in the "create group chat" task, so we can take profit of some methods there and test if the toolbar name is the channel expected name, if the chat history title corresponds with the channel name and so on!

File: test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py [hunk 5]
```diff
@@ -25,13 +25,13 @@ def step(context, community_name):
     StatusCommunityScreen()
     _statusCommunityScreen.verify_community_name(community_name)
     
-@When("the admin creates a community channel named |any|, with description |any|")
-def step(context, community_channel_name, community_channel_description):
-    _statusCommunityScreen.create_community_channel(community_channel_name, community_channel_description)
+@When("the admin creates a community channel named |any|, with description |any| with the method |any|")
```
Comment (noeliaSD) [inline test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py:28]: Another comment related to validation.. maybe we can add some validation regarding the admin user.. Can we verify the community channel is created BY THE ADMIN??

File: test/ui-test/src/screens/StatusCommunityScreen.py [hunk 6]
```diff
@@ -37,8 +38,14 @@ def __init__(self):
     def verify_community_name(self, communityName: str):
         verify_text_matching(CommunityScreenComponents.COMMUNITY_HEADER_NAME_TEXT.value, communityName)
     
-    def create_community_channel(self, communityChannelName: str, communityChannelDescription: str):
-        click_obj_by_name(CommunityScreenComponents.COMMUNITY_CREATE_CHANNEL_OR_CAT_BUTTON.value)
+    def create_community_channel(self, communityChannelName: str, communityChannelDescription: str, method: str):
+        if (method == "bottom_menu"):
```
Comment (noeliaSD) [inline test/ui-test/src/screens/StatusCommunityScreen.py:42]: Could it make sense to use an enum here instead of a string?

**Review by noeliaSD — APPROVED** [review]
LGTM!

Some minor comments that you can decide if you want to address here or in a different discuss it in a different task!

File: test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py (hunk 5 above)
Comment (jrainville) [reply test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py:28]: I'm not sure. Technically, a user could not create a channel, so we can do another test later where a user joins a community and test if they can create a channel (they shouldn't be able to).

**Pull request merged**